zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/FlowNodeInstanceQueryController.java

```diff
@@ -45,6 +49,28 @@ public ResponseEntity<FlowNodeInstanceSearchQueryResponse> searchFlownodeInstanc
         .fold(RestErrorMapper::mapProblemToResponse, this::search);
   }
 
+  @GetMapping(
+      path = "/{flowNodeInstanceKey}",
+      produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_PROBLEM_JSON_VALUE})
+  public ResponseEntity<FlowNodeInstanceItem> getByKey(
+      @PathVariable("flowNodeInstanceKey") final Long flowNodeInstanceKey) {
+    try {
+      return ResponseEntity.ok()
+          .contentType(new MediaType(MediaType.APPLICATION_JSON, StandardCharsets.UTF_8))
```

Comment:
🔨  I was reviewing this and discussing with Tobias, It is not necessary set explicit the content type. This can be removed.